src/config/init.rs

```diff
@@ -0,0 +1,39 @@
+use colored::Colorize;
+
+use crate::{models::ResultWithDefaultError, utilities};
+
+pub struct ConfigInitCommand;
+
+impl ConfigInitCommand {
+    pub async fn execute(edit: bool) -> ResultWithDefaultError<()> {
+        let config_path = super::locate::locate_config_path();
+        match config_path {
+            Ok(path) => {
+                if edit {
+                    utilities::open_path_in_editor(path)?
+                } else {
+                    println!(
+                        "{} {}",
+                        "Config file already exists at".yellow(),
+                        path.display()
+                    );
+                    return Ok(());
+                }
+            }
+            Err(_) => {
+                let default_config = include_bytes!("./fixtures/default.toml");
+                let config_path = super::locate::get_config_path_for_current_dir()?;
+                let config_dir = config_path.parent().unwrap();
+                let msg = format!(
+                    "{} {}",
+                    "Created config at".green().bold(),
+                    config_path.display()
```

Comment:
Maybe shorten this to `~/.toggl-cli/path` on Unix and `%localappdata%/.toggl-cli/path` on windows